client/my-sites/patterns/types.ts

```diff
@@ -76,7 +76,10 @@ export type ReadymadeTemplate = {
 	template_id: number;
 	title: string;
 	description: string;
-	content: string;
-	screenshot: string;
+	home: {
+		header: string;
+		content: string;
+		footer: string;
+	};
```

Comment:
We could have the `ReadymadeTemplateHome` type to avoid duplicating this on client/landing/stepper/declarative-flow/readymade-template.tsx